applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import {
+  makeStyles,
+} from "@material-ui/core";
+import { Button, Dialog, DialogActions, DialogContent,  DialogTitle, IconButton, Box, Typography } from "@material-ui/core";
+import {orange} from  "../../theme"
+import CLOSE from "../../assets/images/icons/close.svg";
+
+const useStyles = makeStyles(() => ({
+    info: {
+        '& .MuiTypography-root': {
+            fontSize: '0.75rem',
+            lineHeight: '1rem',
+            letterSpacing: '0.001em',
+          },
+    }, 
+
+}));
+
+
+
+interface IDeleteDialog {
+    open:boolean;
+    handleClose: () => void
+}
+
+export const DeleteExperimentDialog: React.FC<IDeleteDialog> = ({open, handleClose}) => {
+  const classes = useStyles()
+
+  return (
```

Comment:
@paularah you can reuse the already existing Basedialog, i know it has slight different UI like the button color, shadows and padding, that can be easily handles with a class